src/__tests__/CompareResults/fetchCompareResults.test.tsx

```diff
@@ -74,8 +73,7 @@ describe('Results View/fetchCompareResults', () => {
       />,
     );
 
-    expect(store.getState().compareResults.data).toStrictEqual({});
```

Comment:
I removed the expectations related to the redux store, because this was testing internal states instead of things seen by the user. But I did add some expectations related to what the user sees.